packages/coding-style/tests/Rector/ClassMethod/AnnotateThrowablesRector/Fixture/custom_exception_already_annotated.php.inc

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Rector\CodingStyle\Tests\Rector\ClassMethod\AnnotateThrowables\Fixture;
+
+class CustomExceptionAlreadyAnnotated
+{
+    /**
+     * @throws \Rector\CodingStyle\Tests\Rector\ClassMethod\AnnotateThrowables\Source\TheException
+     */
+    public function throwException()
+    {
+        throw new \Rector\CodingStyle\Tests\Rector\ClassMethod\AnnotateThrowables\Source\TheException();
```

Comment:
Add also new test case fixture with:

```php
throw $value
```

Reply:
Read this: https://github.com/rectorphp/rector/pull/2833#issuecomment-584948298